core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -497,6 +500,37 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
     }
   }
 
+  private def getExistingWorkspaceInvites(workspaceName: WorkspaceName) = {
```

Comment:
`getExistingWorkspaceInvites`, `deleteWorkspaceInvites`, and `saveWorkspaceInvites` are all only called from inside `updateACL`. Other cases like this have the functions defined inside `updateACL` instead of being private on `WorkspaceService`. Is there a reason for these to be different or could they be changed to use consistent visibility? (I'm not sure which is preferred.)